### PR TITLE
Fix English Translation

### PR DESCRIPTION
**Pull request by godly-devotion**
Thank you for Mos! I really appreciate it.
I found and fixed some minor spelling and grammar mistakes.

### CODE REVIEW

**Comment by Caldis** [comment]
Awesome @godly-devotion, you corrected so many mistakes !
I'll release it soon. Thank you for your contribution !

File: Mos/en.lproj/Main.strings [hunk 1]
```diff
@@ -7,48 +7,48 @@
...
+"vmk-zZ-ifN.title" = "Sets the duration of the scroll animation";
+"2ho-yL-X2k.title" = "Reset to defaults";
+"bAe-Zc-5SS.title" = "In some applications, scrolling is used to adjust certain precise parameters (such as adjusting the volume, zooming the display, etc.) and smooth scrolling may cause interference. You can set a block key which will temporarily disable smooth scrolling when the key is pressed.";
 // Exception
-"I5j-Bv-Aiy.title" = "Need to using Accessibility features";
-"FU4-wI-K8f.title" = "To identify the exception application, we need to dynamically obtain the window information. Therefore, please allow us access to your accessibility feature.";
+"I5j-Bv-Aiy.title" = "Needs access to Accessibility feature";
+"FU4-wI-K8f.title" = "In order to custom the scroll behavior in certain applications, please allow access to Accessibility feature.";
```
Comment (godly-devotion) [inline Mos/en.lproj/Main.strings]: I misssed this but this should say "In order to customize..." not "custom". Do you want me to create a different pull request or would you like to fix it?

Comment (Caldis) [reply Mos/en.lproj/Main.strings]: Very welcome you to continue to create a PR !  😀😁

File: Mos/Localization/Localizable.swift [hunk 2]
```diff
@@ -5,29 +5,28 @@
 //  Created by Caldis on 2018/2/25.
 //  Copyright © 2018年 Caldis. All rights reserved.
 //
-
 import Cocoa
 
 class i18n {
-    
+
```
Comment (wyudong) [inline Mos/Localization/Localizable.swift]: Do not change author's code style, just follow it. Propose a better style in another commit if necessary.

File: Mos/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -17,9 +17,9 @@
 "Quit" = "Quit";
 "Hide Icon" = "Hide Icon";
 
-// Notifaction
+// Notification
 "Mos is running in the background" = "Mos is running in the background";
-"Mos's status bar icon is hidden" = "Mos's status bar icon is hidden";
+"Mos' status bar icon is hidden" = "Mos' status bar icon is hidden";
```
Comment (wyudong) [inline Mos/Localization/en.lproj/Localizable.strings]: "Mos status bar icon is hidden" is enough.

File: Mos/en.lproj/Main.strings [hunk 4]
```diff
@@ -7,48 +7,48 @@
...
+"NAp-3R-e8q.title" = "Toggle Key:";
+"lzC-ub-pqJ.title" = "Change vertical scrolling to horizontal scrolling";
 "akk-dX-tW4.title" = "Block Key:";
-"62p-76-okJ.title" = "Temporarily block the smoothing of the scroll";
+"62p-76-okJ.title" = "Temporarily block smooth scrolling";
 "bcp-Xx-rdX.title" = "Disable";
 "DXO-Xo-tJd.title" = "Disable";
 "YE2-UH-9Ku.title" = "Step:";
-"3Xk-SU-gqO.title" = "The minium scroll distance.";
+"3Xk-SU-gqO.title" = "Sets the minimum scroll distance";
```
Comment (wyudong) [inline Mos/en.lproj/Main.strings]: Use "set" not "sets" here.

File: Mos/en.lproj/Main.strings [hunk 5]
```diff
@@ -7,48 +7,48 @@
...
-"vmk-zZ-ifN.title" = "The duration of scroll animation";
-"2ho-yL-X2k.title" = "Reset to default";
-"bAe-Zc-5SS.title" = "In some applications, scrolling is used to adjust certain precise parameters (such as adjusting the volume or zooming the document, etc.), and smooth scrolling may cause some interference to the operation. You can set a block key to perform such operations. Temporarily blocking the smoothing of the scroll.";
+"vmk-zZ-ifN.title" = "Sets the duration of the scroll animation";
+"2ho-yL-X2k.title" = "Reset to defaults";
+"bAe-Zc-5SS.title" = "In some applications, scrolling is used to adjust certain precise parameters (such as adjusting the volume, zooming the display, etc.) and smooth scrolling may cause interference. You can set a block key which will temporarily disable smooth scrolling when the key is pressed.";
 // Exception
-"I5j-Bv-Aiy.title" = "Need to using Accessibility features";
-"FU4-wI-K8f.title" = "To identify the exception application, we need to dynamically obtain the window information. Therefore, please allow us access to your accessibility feature.";
+"I5j-Bv-Aiy.title" = "Needs access to Accessibility feature";
```
Comment (wyudong) [inline Mos/en.lproj/Main.strings]: Same as "set".

File: Mos/en.lproj/Main.strings [hunk 6]
```diff
@@ -7,48 +7,48 @@
...
+"FU4-wI-K8f.title" = "In order to custom the scroll behavior in certain applications, please allow access to Accessibility feature.";
 "Pw5-2q-7Fa.title" = "Allow to use accessibility";
 "bga-wG-iim.title" = "Exception";
-"Rgu-cb-5lp.title" = "Exceptional Applications:";
+"Rgu-cb-5lp.title" = "Customize per Application:";
```
Comment (wyudong) [inline Mos/en.lproj/Main.strings]: IMO the new translation is wrong. Can be "Excluded applications".

File: Mos/en.lproj/Main.strings [hunk 7]
```diff
@@ -60,11 +60,11 @@
 "yDf-dn-Mcz.title" = "Home Page";
 "99M-8D-HLN.title" = "GitHub";
 "AM8-cj-DnH.title" = "Buy author a cup of coffee";
-"aPi-cE-maK.title" = "Donate by paypal";
+"aPi-cE-maK.title" = "Donate using PayPal";
```
Comment (wyudong) [inline Mos/en.lproj/Main.strings]: [Donate with PayPal](https://www.paypal.com/fundraiser/hub), or via PayPal.

File: Mos/Localization/Localizable.swift (hunk 2 above)
Comment (godly-devotion) [reply Mos/Localization/Localizable.swift]: My XCode is set to remove extra whitespaces on save. As you can see they are not needed.

File: Mos/Localization/en.lproj/Localizable.strings (hunk 3 above)
Comment (godly-devotion) [reply Mos/Localization/en.lproj/Localizable.strings]: Either one can work. The one I used could be arguable more correct as it is possessive saying that it is the status bar icon of Mos.

File: Mos/en.lproj/Main.strings (hunk 4 above)
Comment (godly-devotion) [reply Mos/en.lproj/Main.strings]: Either one works in this case. They are used evenly in everyday English.

File: Mos/en.lproj/Main.strings (hunk 5 above)
Comment (godly-devotion) [reply Mos/en.lproj/Main.strings]: I don't see where "set" would go here.

File: Mos/en.lproj/Main.strings (hunk 6 above)
Comment (godly-devotion) [reply Mos/en.lproj/Main.strings]: If you take a look at the settings, you can actually toggle not only the smooth scroll setting but the reverse scrolling as well. In that way its more than just Excluding applications but closer to Customizing the scroll behavior per application.

File: Mos/en.lproj/Main.strings (hunk 7 above)
Comment (godly-devotion) [reply Mos/en.lproj/Main.strings]: Either works well grammatically speaking.

File: Mos/Localization/Localizable.swift (hunk 2 above)
Comment (Caldis) [reply Mos/Localization/Localizable.swift]: These insignificant changes should be referred to separately as a PR. Bug, this is not a big problem. These spaces are really useless.

File: Mos/en.lproj/Main.strings (hunk 4 above)
Comment (wyudong) [reply Mos/en.lproj/Main.strings]: kidding me?

Comment (godly-devotion) [reply Mos/en.lproj/Main.strings]: This sentence is a simplified version of `This [the slider] sets the minimum scroll distance`. But one can remove the word `this` in the beginning because it is obvious as to what `this` is referring to (the slider). So we can render the sentence as `Sets the minimum scroll distance`. However `This set the minimum scroll distance` is grammatically incorrect.
